src/libraries/System.Security.Cryptography.Cose/tests/CoseHeaderLabelTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+    public class CoseHeaderLabelTests
+    {
+        [Fact]
+        public void CoseHeaderLabel_GetHashCode()
+        {
+            CoseHeaderLabel label = default;
+            Assert.Equal(0.GetHashCode(), label.GetHashCode());
+
+            label = new CoseHeaderLabel();
+            Assert.Equal(0.GetHashCode(), label.GetHashCode());
+
+            label = new CoseHeaderLabel(0);
+            Assert.Equal(0.GetHashCode(), label.GetHashCode());
+
+            label = new CoseHeaderLabel("");
+            Assert.Equal("".GetHashCode(), label.GetHashCode());
+
+            label = new CoseHeaderLabel(1);
+            Assert.Equal(1.GetHashCode(), label.GetHashCode());
```

Comment:
Int32.GetHashCode() is `return this;`  You could remove the redundant word avoid the GitHub syntax highlighter saying there's invalid code here.